alternative-install/docker-compose/images/backend-module/Readme.md

```diff
@@ -0,0 +1,49 @@
+# Dockerfile for backend-modules
+
+With this Dockerfile comes a shell script as `docker-entrypoint` which performs the registration and activation of modules depending on whether there exists a module-descriptor in the public folio-registry and existing okapi-installations.
+
+simply adjust the environment variables below matching your installation parameters and start the container. a typical `docker-compose.yml` would look like this:
+```yml
+  mod-calendar-1.0.2:
+    image: folioorg/mod-calendar:1.0.2
+    environment:
+      SERVICE_URL: http://mod-calendar-1.0.2:8081
+      DB_DATABASE: okapi
+      DB_USERNAME: okapi
+      DB_PASSWORD: changeMe
+      DB_HOST: okapi-db-2.22.0
+      DB_PORT: "5432"
+      OKAPI_URL: http://okapi-2.22.0:9130
+      SERVICE_ID: mod-calendar-1.0.2
+      INSTALL_ID: mod-calendar-1.0.2
+      SELF_DEPLOY: "true"
+    depends_on:
+    - okapi-2.22.0
+    - okapi-db-2.22.0
+  ...
+```
+
+On first startup the module-container will look for *okapi* at the provided OKAPI_URL and perform the registration-process via *curl*. there are multiple retries built in the process to ensure okapi comes up and ready accepting connections.
+
+If the module is registered successfully the JAR is started provided by the command in env var `START_CMD`
+
+## Configuration variables
```

Comment:
Can you extend the title mentioning that the examples are the defaults? `Configuration variables and defaults`